FirebaseUI/Auth/AuthProviderUI/Facebook/Source/FIRFacebookAuthUI.h

```diff
@@ -34,26 +34,26 @@ NS_ASSUME_NONNULL_BEGIN
     @brief The scopes to use with Facebook Login.
     @remarks Defaults to using "email" scopes.
  */
-@property(nonatomic, copy) NSArray<NSString *> *scopes;
+@property(nonatomic, copy, readonly) NSArray<NSString *> *scopes;
```

Comment:
+1 to that view. It may be worth flagging, just in case people have been constructing and then setting scopes.